src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -331,6 +348,22 @@ public function sanitise(HTMLValue $html)
                     $el->setAttribute($attr, $forced);
                 }
             }
+
+            // add rel="noopener, noreferrer" to link elements with a target attribute
+            if ($linkNooperEnabled && $el->tagName === 'a') {
```

Comment:
There's a lot of nesting here, are you able to reduce it a little? I think firstly adding a private method like `$this->addRelValue($el)` would be a good start, then you can utilise early returns instead of else and elseifs

Reply:
Have moved to a private function to reduce nesting, though I kept the elseifs.  I'm not sure how early returns would for the ifelseif block so I've left it as is